### ISSUE TITLE
MolGan tutorial gives NAN

### ISSUE THREAD

**Issue by mghahremanpour** [keras-team/keras-io#2049]
Hi,

I am running the MolGan model provided at https://github.com/keras-team/keras-io/blob/master/examples/generative/molecule_generation.py, but training gives NaN when more than one convolution layer is used in the encoder. I was wondering if you could help me with this. 

Thanks

**Comment by sachinprasadhs** [comment]
Hi, If you are modifying the existing code, could you please provide the code Gist to replicate the reported behavior. Thanks

**Comment by mghahremanpour** [comment]
Code is not modified!

**Comment by sachinprasadhs** [comment]
@mghahremanpour , Could you point us to the part where you are seeing NaN

**Comment by mghahremanpour** [comment]
The loss value becomes NaN with more than 2 conv layers in get_encoder
